cmd/zfs_object_agent/zettaobject/src/pool.rs

```diff
@@ -1060,29 +1090,26 @@ impl Pool {
             }
         }
 
-        // XXX wait for them all at once
-        for log in syncing_state.pending_frees_log.iter_mut() {
-            log.flush(txg).await;
+        // XXX wait for them all at once using
```

Comment:
```suggestion
        // XXX wait for them all at once using FuturesUnordered
```
See cleanup_log_objects() for an example of how to do this.